src/pages/home/report/ReportActionsList.js

```diff
@@ -78,10 +78,10 @@ function keyExtractor(item) {
 const ReportActionsList = (props) => {
     const opacity = useSharedValue(0);
     const animatedStyles = useAnimatedStyle(() => ({
-        opacity: withTiming(opacity.value, {duration: 100}),
```

Comment:
@terrysahaidak Can you please tell me why is this change necessary? I don't recall any recent issues related to using `withTiming` inside `useAnimatedStyle`. Have you found some bug in Reanimated?

Reply:
No, it's not related to a bug or anything, just a more common on obvious in my opinion way of running animation - assigning animation to a shared value.